content/_data/solutions/bitbucketserver.yml

```diff
@@ -0,0 +1,10 @@
+usecase: "Bitbucket Server"
+blog_tags:
+- bitbucket
+articles:
+
+plugins:
+-
```

Comment:
Is this supposed to be here?

Reply:
@timja that was a mistake. Thanks for letting me know.